docs/howto/system_testing.md

```diff
@@ -0,0 +1,140 @@
+# HOWTO: Writing system tests for a package
+
+## Introduction
+Elastic Packages are comprised of data streams. A system test exercises the end-to-end flow of data for a package's data stream — from ingesting data from the package's integration service all the way to indexing it into an Elasticsearch data stream.
+
+## Process
+
+Conceptually, a system test will perform the following steps:
+1. Deploy the Elastic Stack, including a 1-node Elasticsearch cluster, a Kibana instance, and an instance of Elastic Agent.
+1. Enroll the Elastic Agent with Fleet (running in the Kibana instance).
```

Comment:
Actually these steps are not part of the test runner. The stack must be up before.

Reply:
So this section is about the _conceptual_ steps involved in system testing. Further down in the _Running a System Test_ section I do explain that the stack must be up before independently of running the system test.

Reply:
I saw you described it at the bottom, but wasn't sure if the reader assumes that the test runner should boot up the stack. I'm fine with leaving this as is.

Reply:
I tried to clarify this a bit while also leaving the conceptual instructions in place.

Reply:
Updated in 0716a71a02f707764152dbd5e69e9fb57adafcc7.